content/en/docs/concepts/storage/ephemeral-volumes.md

```diff
@@ -125,8 +125,8 @@ driver. These attributes are specific to each driver and not
 standardized. See the documentation of each CSI driver for further
 instructions.
 
-As a cluster administrator, you can use a [PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/) to control which CSI drivers can be used in a Pod, specified with the
-[`allowedCSIDrivers` field](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podsecuritypolicyspec-v1beta1-policy).
+As a cluster administrator, you can use [Pod SecurityPolicy](/docs/concepts/security/pod-security-policy/)` to control which CSI drivers can be used in a Pod, specified with the
+[`allowedCSIDrivers` field](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podsecuritypolicyspec-v1beta1-policy). Take note that the PodSecurityPolicy API is **deprecated**.
```

Comment:
```suggestion
As a cluster administrator, you can use [Pod SecurityPolicy](/docs/concepts/security/pod-security-policy/) to control which CSI drivers can be used in a Pod, specified with the
[`allowedCSIDrivers` field](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podsecuritypolicyspec-v1beta1-policy). Take note that the PodSecurityPolicy API is **deprecated**.
```
The link to `allowedCSIDrivers` field does not render properly because of the extra `